client/src/components/Heroes/HeroesRadarTab.tsx

```diff
@@ -19,25 +26,53 @@ const initialPage = 1;
 const initialPageSize = 20;
 const initialQueryParams = { current: initialPage, pageSize: initialPageSize };
 
+const { RangePicker } = DatePicker;
+
+const rangePresets: TimeRangePickerProps['presets'] = [
+  { label: 'Last 7 Days', value: [dayjs().add(-7, 'd'), dayjs()] },
```

Comment:
please reuse `dayjs()` for optimization purposes

Reply:
fixed